x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -28,7 +28,8 @@ import { JOB_MAP_NODE_TYPES } from '../../../../../../common/constants/data_fram
 // import { DeleteButton } from './delete_button';
```

Comment:
Should we get rid of this commented import?

Reply:
Seems like the deletion functionality will be added in a follow up. I think if we don't want to remove it, it would be good to add a todo flag.

Reply:
Good call - added in 0e95f8993b8184dfdf3f02eaad3b476eda9ac79a